packages/checkup-plugin-ember/src/results/ember-test-types-task-result.ts

```diff
@@ -0,0 +1,47 @@
+import { BaseTaskResult, TaskResult, ui, PieChartData, TaskItemData } from '@checkup/core';
+
+export default class EmberTestTypesTaskResult extends BaseTaskResult implements TaskResult {
+  testTypes!: TaskItemData[];
+
+  findByType(typeName: string): TaskItemData | undefined {
+    return this.testTypes.find((type) => type.type === typeName);
+  }
+
+  toConsole() {
+    ui.styledHeader(this.meta.friendlyTaskName);
+    ui.blankLine();
+    ui.table(this._deriveTestPercentage(this.testTypes), { type: {}, total: {}, percentage: {} });
```

Comment:
I've been using the object argument for the headers to provide a capitalized header string for more consistent output. Can we do that?

```suggestion
    ui.table(this._deriveTestPercentage(this.testTypes), { type: { header: 'Test Type' }, total: { header: 'Total' }, percentage: { header: '' } });
```

I left the last blank as I'm not sure we need a header in that column?

Reply:
the ui table already capitalizes the headers... if you look at the screenshot of the output I have above, they are already capitalized

Reply:
Oh so it does. I wonder why it wasn't capitalizing on my task. Will have to check.